control/velocity_controller_lqr/package.xml

```diff
@@ -0,0 +1,28 @@
+<?xml version="1.0"?>
+<?xml-model href="http://download.ros.org/schema/package_format3.xsd" schematypens="http://www.w3.org/2001/XMLSchema"?>
+<package format="3">
+  <name>velocity_controller_lqr</name>
+  <version>1.0.0</version>
+  <description>Velocity controller package for the AUV Orca</description>
+  <maintainer email="cyprian.github@gmail.com">cyprian</maintainer>
+  <license>MIT</license>
+
+  <buildtool_depend>ament_cmake</buildtool_depend>
+
+  <depend>rclcpp</depend>
+  <depend>rclpy</depend>
+  <depend>geometry_msgs</depend>
+  <depend>nav_msgs</depend>
+  <depend>vortex_msgs</depend>
+
+  <exec_depend>control</exec_depend>
```

Comment:
You need to replace ```<exec_depend>control</exec_depend>``` with the correct rosdep key for control. I think the correct one is ```<depend>python-control-pip</depend>```

Reply:
![image](https://github.com/user-attachments/assets/d44dead9-ab34-41c5-bfd4-207f5b9173f0)

Reply:
najs <depend>python-control-pip</depend> implemented chief